.github/workflows/build.yml

```
@@ -62,7 +62,7 @@ jobs:
       - id: setup-python
         uses: actions/setup-python@v5
         with:
-          python-version: ${{ steps.setup-env.outputs.python-version }}
+          python-version: "3.12"
```

The version from cisagov/setup-env-github-action [is already 3.12](https://github.com/cisagov/setup-env-github-action/blob/develop/action.yml).  Why have you hardcoded this line?

```suggestion
          python-version: ${{ steps.setup-env.outputs.python-version }}
```

---

I don't think it was `3.12` when I cloned.  🤷 (maybe)

In any case.  Fixed in:
- 0f917aeb84c953c25cf412a0eabce627aca6f7b3